External/FEXCore/Source/Interface/Core/JIT/Arm64/MemoryOps.cpp

```diff
@@ -665,8 +705,8 @@ void JITCore::RegisterMemoryHandlers() {
     REGISTER_OP(STOREMEMTSO,         StoreMemTSO);
   }
   else {
-    REGISTER_OP(LOADMEMTSO,          LoadMem);
-    REGISTER_OP(STOREMEMTSO,         StoreMem);
+    REGISTER_OP(LOADMEMTSO,          Unhandled);
```

Comment:
Same comment as from the x86 side.
Maybe even further that if you are expecting to have TSO ops entirely eliminated prior to them hitting the backend, then we should have the TSO ops never be conditional selecting the handling and instead always act as TSO; Regardless of the config option.

Reply:
Yeah, the only reason I changed this was because it was conditional before. I do like the validation aspect, but that can be moved to the IR validation passes (if we want, when we have free time). I'll change it for now to not be conditional across all backends ~

Reply:
Done